spec/controllers/autocomplete_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'set'
+RSpec.describe AutocompleteController, type: :controller do
+  let(:organization) { GitHubFactory.create_owner_classroom_org }
+  let(:user)         { organization.users.first                 }
+
+  before do
+    sign_in(user)
+  end
+
+  describe 'GET #github_repos', :vcr do
+    it 'returns success and returns a subset of repos of the user' do
```

Comment:
best practice here is to have a single `expect` per `it` block. That way the test output is more descriptive when something fails.